showkase-browser-testing/src/androidTest/java/com/airbnb/android/showkase_browser_testing/ShowkaseBrowserTest.kt

```diff
@@ -816,4 +816,36 @@ class ShowcaseBrowserTest {
 
         }
     }
+
+    @Test
+    fun customAnnotatedPrivateComposablesShouldNotShot() {
```

Comment:
I was wondering what `ShouldNotShot` implies here?

Reply:
Ah yeah, good point! I think I misspelled to `ShouldNotShow` here, but I can change it and add a comment maybe? :)